redis/src/cluster_async/mod.rs

```diff
@@ -461,17 +470,16 @@ impl<C> Future for Request<C> {
                         self.respond(Err(err));
                         return Next::Done.into();
                     }
+                    OperationTarget::NoTargetFound => {
+                        warn!("No connection found: `{err}`");
```

Comment:
can you also log the RequestInfo

Reply:
What part of the request info? We don't log commands, because they might contain PPI.